### PR TITLE
github: add pull request template

### PR DESCRIPTION
**Pull request by giacope**
**What does this PR do, and why do we need it?**

So that we get "this" information on every PR.

**Which issue does this PR fix?**

None.

**Special notes for the reviewers**:

This is an example of how the Pull Request template will look like.

### CODE REVIEW

**Review by tmacro — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+Thank you for contributing to Zenko!
+
+Please fill the information below if applicable.
```
Comment (wabernat) [inline .github/PULL_REQUEST_TEMPLATE.md]: Please enter applicable information below.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+-->
+
+**What this PR does and why do we need it**:
```
Comment (wabernat) [inline .github/PULL_REQUEST_TEMPLATE.md]: What does this PR do, and why do we need it?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+**Which issue this PR fixes**:
```
Comment (wabernat) [inline .github/PULL_REQUEST_TEMPLATE.md]: Which issue does this PR fix?

**Review by anurag4DSB — CHANGES_REQUESTED** [review]
Great Idea
I was looking at the S3 pull request template.
It would be great to add things from this too:
https://github.com/scality/cloudserver/blob/a65eb808734c056f959f09e72c3cf48414a52f6b/.github/PULL_REQUEST_TEMPLATE.md

Reason: It has scality guidelines too.

**Comment by anurag4DSB** [comment]
at least have scality Contribution guidelines link: In order to contribute to the project, you must sign your work https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#sign-your-work

**Comment by giacope** [comment]
@anurag4DSB IMHO, the Cloudserver template doesn't look like a "template" and it's more like a summary of guidelines. Also, most of the guidelines are outdated and irrelevant in the context of Zenko, so I think it's pointless to reference them.
Finally, I think it will be hypocritical to ask people to sign commits under the DCO when none of us does it. If it wants to be enforced, then it should be enforced and not just mentioned. This can be done through a GitHub bot.

**Review by ssalaues — APPROVED** [review]

**Pull request merged**